src/MassReply.res

```diff
@@ -72,14 +72,17 @@ let make = (~conversations, ~onMassReplySent) => {
       {textEl(`Hinweis: Die Empänger der Nachricht sehen nicht, dass es sich um eine Sammelantwort handelt.`)}
     </p>
     <div className="recipient-list">
-      <div> <strong> {textEl(`Empfänger:`)} </strong> </div>
+      <div>
+        <strong> {textEl(`Empfänger:`)} </strong>
```

Comment:
Previously one had to use template strings to represent umlauts. Now in ReScript 11 all string representations are utf8 compatible (so `"Empfänger"` will be equivalent).

Didn't change it here yet, but just so you know.